docs/getting_started/get_started.md

```diff
@@ -132,3 +132,15 @@ such as preferred timezone, short and full date and time format, or Back Office
 --------|-----------|
 |Active dashboard|Sets which dashboard is displayed after you log in.|
 
+### Change the password
+
+You can change your user [password]([[= developer_doc =]]/users/passwords/) at any time.
```

Comment:
```suggestion
You can change your user [password]([[= developer_doc =]]/users/passwords/) at any time.
```

I would add a disclaimer here: "If the administrator granted you the right permissions, ..."

Reply:
It's explained in the "Passwords" page mentioned in link.